packages/cli/src/cmds/init/initUtils.ts

```diff
@@ -336,3 +345,29 @@ export async function prepareWorkingDirectory(): Promise<void> {
     }
   }
 }
+
+/**
+ * In case when TypeScript version is used, default source file
+ * will be updated if previous was not correct.
+ *
+ * @param yargsArgv - The Yargs arguments object.
+ * @returns Modified Yargs arguments object.
+ */
+export function correctDefaultArgs(yargsArgv: Arguments): Arguments {
+  if (yargsArgv.template === 'typescript' && yargsArgv.src === 'src/index.js') {
+    yargsArgv.src = 'src/index.ts';
+    yargsArgv.s = 'src/index.ts';
+  }
+
+  return yargsArgv;
+}
+
+/**
+ * Check if template argument is 'typescript'.
+ *
+ * @param templateType - String value of the template parameter passed from CLI.
+ * @returns True or false.
+ */
+export function isTemplateTypescript(templateType: any): boolean {
```

Comment:
Please type this argument

Reply:
Added type, but will have to use `as string` when calling it, because `argv.template` doesn't know about the type.